src/MultiFactoryCollection.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Christophrumpel\LaravelFactoriesReloaded;
+
+use Illuminate\Support\Collection;
+
+class MultiFactoryCollection{
+    protected Collection $factories;
+
+    public function __construct(Collection $factories)
+    {
+        $this->factories = $factories;
+    }
+
+    public function create()
```

Comment:
Can you please add return types to "create" and "make" method.

Reply:
Whoops. Done.